cmd/antrea-agent/agent.go

```diff
@@ -162,6 +162,7 @@ func run(o *Options) error {
 	_, ipsecAuthenticationMode := config.GetIPsecAuthenticationModeFromStr(o.config.IPsec.AuthenticationMode)
 	networkConfig := &config.NetworkConfig{
 		TunnelType:            ovsconfig.TunnelType(o.config.TunnelType),
+		TunnelPort:            ovsconfig.TunnelPort(o.config.TunnelPort),
```

Comment:
nit: it may be simpler to use just int as the port's type to avoid many type conversions. And given port is actually 16bits, int32 is not helpful too. If we want to validate, it could just check the value in `validate()`.

Reply:
Thanks for the review, updated.